x-pack/plugins/watcher/public/components/threshold_watch_expression/lib/agg_types.js

```diff
@@ -5,33 +5,44 @@
  */
 
 import { AGG_TYPES } from 'plugins/watcher/../common/constants';
+import { i18n } from '@kbn/i18n';
 
 export const aggTypes = {
   count: {
-    label: 'count()',
+    label: i18n.translate('xpack.watcher.thresholdWatchExpression.aggTypes.countLabel', {
+      defaultMessage: 'count()',
```

Comment:
I'm not sure whether we should translate this labels. Could you please provide a screenshot with this labels on UI?

Reply:
there
![image](https://user-images.githubusercontent.com/25555662/47208782-630b2f80-d397-11e8-9a93-3d4a1b6421f1.png)

Reply:
@Nox911 we shouldn't translate function names. Please revert translations in this file.